subprojects/unit-system.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-git]
```

Comment:
wrap-git is not permitted in the wrapdb, please use the `${repo}/archive/${gitref}.tar.gz` tarball